front/src/applications/stdcm/views/StdcmRequestModal.tsx

```diff
@@ -123,11 +124,14 @@ export default function StdcmRequestModal(props: StdcmRequestModalProps) {
         .catch((e) => {
           // Update simu in redux with data;
           setCurrentStdcmRequestStatus(STDCM_REQUEST_STATUS.rejected);
-
           dispatch(
             setFailure({
               name: t('operationalStudies/manageTrainSchedule:errorMessages.stdcmError'),
-              message: e?.response?.data?.message, // axios error, def is ok
```

Comment:
Not necessarily, the request may be somehow invalid.

There's also the possibility that there's no path at all (like when looking for a path between two points that aren't connected). But I don't think it's possible to tell this case apart without changes in core. 

Plus, it seems that editoast forwards failed assertion errors as 400 (I thought that was fixed it doesn't seem so, on `3ec3e4ad2c60bcce386e35df51e775c91cae64c4`)

Reply:
I added a test on the message, is it ok for you ?